osf_tests/test_region.py

```diff
@@ -0,0 +1,68 @@
+import mock
+import pytest
+
+from framework.auth import Auth
+
+from osf.models import MetaSchema, Registration
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    RegionFactory
+)
+
+
+@pytest.mark.django_db
+class TestRegion:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
```

Comment:
Minor: Since you're not using `user.auth`, you can use `UserFactory()`, which is faster to create than `AuthUserFactory()` becauase it doesn't set a password for the user.